index.html

```diff
@@ -9,36 +9,214 @@
     <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" >
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;600&display=swap" rel="stylesheet" crossorigin="anonymous" >
 </head>
 <body>
   <main>
-      <div class="header">
-       <a href="">My Logo</a>
-      <nav><a href=""><i class="fas fa-bars"></i></a></nav>
-      </div> 
+    <div class="header">
+      <a href="">My Logo</a>
+      <nav><a href="" aria-label="bars"><i class="fas fa-bars"></i></a></nav>
+    </div>
+
     <section class="container">
       <div class="intro">
         <h1>Hi! I'M DIRE</h1> 
           <h1>GLAD TO SEE <br>YOU!</h1>
-         <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+          <p>I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience!
+             If you like what you see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+      </div>
+
+      <div class="contact"> 
+        <a href="#" class="contacts">LET'S CONNECT</a>
       </div>
 
-    
-       <div class="contact"> 
-          <a href="#" class="contacts">LET'S CONNECT</a>
-        </div> 
       <div class="social-media">
         <ul class="sm">
-            <li><a href="#"><i class="fab fa-twitter"></i></a></li> 
-            <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
-            <li><a href="#"><i class="fab fa-medium-m"></i></a></li>
-            <li><a href="#"><i class="fab fa-github"></i></a></li> 
+          <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+          <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+          <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+          <li><a href="#" aria-label="github"><i class="fab fa-github"></i></a></li> 
         </ul>
       </div>
     </section>
+      
+    <section>
+      <ul class="cards-container">
+        <li class="work one">
```

Comment:
- [ ] For each project ("work") content of the project is aligned with flexbox. Please kindly implement that.